cpuinfo.go

```diff
@@ -11,9 +11,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build linux
-// +build linux
```

Comment:
Removing this will likely break other non-linux non-darwin OSs like BSDs, etc.

Reply:
I was worried about that too but couldn't find a better solution. Will you be able to suggest better build tags?